frappe/tests/test_modules.py

```diff
@@ -0,0 +1,162 @@
+import os
+import shutil
+import unittest
+
+import frappe
+from frappe import scrub
+from frappe.core.doctype.doctype.test_doctype import new_doctype
+from frappe.custom.doctype.custom_field.custom_field import create_custom_field
+from frappe.model.meta import trim_table
+from frappe.modules import export_customizations, export_module_json, get_module_path
+from frappe.modules.utils import export_doc, sync_customizations
+from frappe.utils import now_datetime
+
+
+def write_file(path, content):
+	with open(path, "w") as f:
+		f.write(content)
+
+
+def delete_file(path):
+	os.remove(path)
+
+
+def delete_path(path):
+	shutil.rmtree(path)
+
+
+class TestUtils(unittest.TestCase):
+	def setUp(self):
+		if self._testMethodName == "test_export_module_json_no_export":
```

Comment:
Does it even make sense to have `setUp` if it's doing something specifically for most of the test 👀 

IMO this code can just be part of test. Test specific teardown can be added with `self.addCleanup`